test/unit/mdc-top-app-bar/util.test.js

```diff
@@ -40,3 +40,15 @@ test('applyPassive returns false for browsers that do not support passive event
   };
   assert.isFalse(util.applyPassive(mockWindow, true));
 });
+
+test('applyPassive returns previous value when called twice without refresh', () => {
+  const mockWindow = {
+    document: {
+      addEventListener: function(name, method, options) {
+        return options.passive;
+      },
+    },
+  };
+  util.applyPassive(mockWindow, true);
+  assert.deepEqual(util.applyPassive(mockWindow, false), {passive: true});
```

Comment:
This is kind of tangential to this PR anyway, but is this really testing what we need it to test? I think this would pass whether we called it with force=true or false. (Also force=true mainly exists for tests to begin with.)

Reply:
I added this to improve branch coverage to 100%. However, after referencing `mdc-ripple` I'll add `applyPassive` to the deregister adapter method so that it gets his twice in a test.